jupyter/Scale Output Example.ipynb

```diff
@@ -0,0 +1,182 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Simple Scalar Output Example\n",
+    "\n",
+    "In this example, we will define an integer parameter called `Integer input` and will output a floating point called `My scalar output` that can be bound to a numeric dashboard tile."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scrapbook as sb \n",
+    "from random import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Parameters\n",
+    "\n",
+    "The parameters block is where you define all of the inputs and outputs to the notebook.  If you look at __Property Inspector >> Advanced Tools >> Cell Metadata__, while the `# Parameters` code cell block is selected, you will see the json that describes the inputs and outputs. Note that when you edit that json, you must click the small check mark under \"Cell Metadata\" to save the changes. In this example you can see we have configured a numeric input and a numeric output."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "papermill": {
+     "parameters": {
+      "numeric_param": 1
+     }
+    },
+    "systemlink": {
+     "namespaces": [
+      "ni-testmanagement"
+     ],
+     "outputs": [
+      {
+       "display_name": "Input plus fraction",
+       "id": "scalar_output",
+       "type": "scalar"
+      }
+     ],
+     "parameters": [
+      {
+       "display_name": "Number",
+       "id": "numeric_param",
+       "type": "number"
+      }
+     ],
+     "version": 2
+    },
+    "tags": [
+     "parameters"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# Parameters\n",
+    "numeric_param = 1 # default value used when input is not provided by Dashboard\n",
+    "\n",
+    "result = [] # container for all of the notebooks results"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "output = int(numeric_param) + uniform(0,1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Results\n",
+    "\n",
+    "Scalar results are appended with a json map that corresponds to the json in the cell metadata described above. The \"type' and 'id' fields should match the cell metadata values, and the 'value' field should contain the scalar result. The config block doesn't correspond to anything in the cell metadata."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Scalar result needs to be added to the result list and glued to the scrapbook\n",
+    "# The json descriptor should match the descriptor in the outputs section in the parameters cell metadata.\n",
+    "result.append({\n",
+    "    'type': 'scalar',\n",
```

Comment:
Via a really roundabout way, the styleguide also recommends double-quotes for Python strings. Technically it just recommends any style rule that `black` also specifies: https://ni.github.io/python-styleguide/#F-1-1 and `black` specifies double quotes. We should probably start using double quotes for new examples - I'm aware that some older product code does not yet follow the newer conventions.

This is another example of the styleguide not perfectly applying to notebooks, as `black` doesn't work on notebooks natively. However, there are ways of getting this to be a thing, ping me if you're particularly curious about python linting.